store/src/elements.rs

```diff
@@ -334,24 +334,31 @@ fn json_to_ops(
       }
     },
     OpType::Receive => {
-      let qty = match ctx_str[..] {
-        ["production", "produce"] => data["qty"].number_or_none(),
-        _ => data["qty"]["number"].number_or_none(),
+      let qty = if data["status"] == "deleted" {
+        None
+      } else {
+        match ctx_str[..] {
+          ["production", "produce"] => data["qty"].number_or_none(),
+          _ => data["qty"]["number"].number_or_none(),
+        }
       };
-      let cost = data["cost"]["number"].number_or_none();
+      let cost =
+        if data["status"] == "deleted" { None } else { data["cost"]["number"].number_or_none() };
 
       if qty.is_none() && cost.is_none() {
-        return Ok(ops);
+        // return Ok(ops);
+        InternalOperation::Receive(0.into(), 0.into())
```

Comment:
why you return something when there is no qty & cost?

Reply:
because I want this operation to be deleted later in mutate_op()